.github/workflows/publish.yml

```diff
@@ -0,0 +1,29 @@
+name: publish
+
+on:
+  push:
+    tags:        
+      - '*'           # Push events to every tag not containing /
+      # Allow manual triggering
+  workflow_dispatch:
+  pull_request:
+    branches: [ '*' ]
```

Comment:
Remove this one please. We don't want a deploy per PR. We only want to check semver there.

Reply:
Of course. It was just for debugging purposes